### PR TITLE
refactor: modal create stream

### PR DESCRIPTION
**Pull request by koverchik**

### CODE REVIEW

**Review by DimaKelek — APPROVED** [review]

File: agoraStreaming/src/Components/ModalCreateStream/index.tsx [hunk 1]
```diff
@@ -29,6 +34,18 @@ export const ModalCreatEvent: FC<ModalCreatEventType> = (props) => {
 
   const newReference = database().ref(`/events/${day}`).push();
 
+  const navigation = useNavigation<StackNavigationPropHome>();
+
+  const createLive = () => {
+    navigation.navigate('Live', {
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Components/ModalCreateStream/index.tsx:40]: please use enum

File: agoraStreaming/src/Components/ModalCreateStream/index.tsx [hunk 2]
```diff
@@ -29,6 +34,18 @@ export const ModalCreatEvent: FC<ModalCreatEventType> = (props) => {
...
+      type: LiveType.CREATE,
+      channelId: uuid(),
+      name: name,
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Components/ModalCreateStream/index.tsx:43]: name: name equal  name

File: agoraStreaming/src/Components/ModalCreateStream/types.ts [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import {DateData} from 'react-native-calendars/src/types';
+import {Region} from 'react-native-maps';
 
 export type ModalCreatEventType = {
-  day: DateData['dateString'];
+  day?: DateData['dateString'];
+  coordinates?: Region;
   isModalVisible: boolean;
   changeModalVisible: () => void;
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Components/ModalCreateStream/types.ts:8]: add empty line and move day props in the end

File: agoraStreaming/src/Screens/Home/index.tsx [hunk 4]
```diff
@@ -132,7 +134,18 @@ export const Home: FC<HomeScreenProps> = () => {
...
-        <ModalCreateChannel coordinates={coordinates} />
+        <ModalCreatEvent
+          changeModalVisible={changeModalVisible}
+          isModalVisible={modalVisible}
+          coordinates={coordinates}
+        />
+        <View style={styles.createContainer}>
+          <TouchableOpacity
+            style={styles.button}
+            onPress={() => setModalVisible(true)}>
```
Comment (evgenyshenets91) [inline agoraStreaming/src/Screens/Home/index.tsx:145]: changeModalVisible

**Pull request merged**